src/components/Form.js

```diff
@@ -147,7 +149,21 @@ const reducer = (draft, action) => {
       }
     });
     history.push('/bevestiging');
-  }
+  };
+
+  const onLogout = async (event) => {
+    event.preventDefault();
+
+    const confirmationQuestion = intl.formatMessage(
+      {id: 'confirmLogout', defaultMessage: 'Are you sure that you want to logout?'}
```

Comment:
I remember reading somewhere that it's not best practice to specify the `id` yourself, but let the extraction tooling deal with that. The example https://formatjs.io/docs/getting-started/message-declaration#using-imperative-api-intlformatmessage also shows that a description is used rather than id.

Reply:
As discussed, removing the `id` causes this issue: https://github.com/formatjs/formatjs/issues/1901
We will remove the ids in the future once we set up the whole translation pipeline. For now, a TODO comment was added next to the messages that use IDs.